src/mxarray.jl

```diff
@@ -159,11 +164,8 @@ const _mx_is_empty   = mxfunc(:mxIsEmpty)
 const _mx_is_struct  = mxfunc(:mxIsStruct)
 const _mx_is_cell    = mxfunc(:mxIsCell) 
 
-
-# getting simple attributes
-
 macro mxget_attr(fun, ret)
-    :( ccall($(fun)::Ptr{Void}, $(ret), (Ptr{Void},), mx.ptr) )
+    :(ccall($(fun)::Ptr{Void}, $(ret), (Ptr{Void},), mx))
```

Comment:
While you are at it (i.e. for another PR maybe). Note that this is exploiting a lowering bug https://github.com/JuliaLang/julia/issues/4873 so the inputs and any reference of variables from calling scope needs to be escaped. It is also a better practice to NOT implicitly use variables in calling scope and pass them in explicitly instead.